.gitignore

```diff
@@ -97,4 +97,5 @@ toc/
 .terraform/*
 .terraform.lock.hcl
 *.tfstate
-*.tfstate.*
\ No newline at end of file
+*.tfstate.*
+*/project.json
```

Comment:
I added a guideline for local configuring, and instructed to add "project.json" file of their target firebase project inside Firestore, where the terraform is. It makes more sense to keep everything needed in one place, instead of updating the "project.json" in config instead.

Reply:
on a second thought, maybe there is no big difference.